Classes/WPEditorView.m

```diff
@@ -1127,7 +1046,7 @@ - (BOOL)isVideoPressInfoRequestScheme:(NSString *)scheme
 
 - (BOOL)isSelectionStyleScheme:(NSString*)scheme
 {
-	static NSString* const kCallbackScheme = @"callback-video-fullscreen";
+	static NSString* const kCallbackScheme = @"callback-selection-style";
```

Comment:
No this must got by mistake, it was another or that is merged in dev already

On Wednesday, 8 April 2015, Diego Rey Mendez <notifications@github.com>
wrote:

> In Classes/WPEditorView.m
> <https://github.com/wordpress-mobile/WordPress-Editor-iOS/pull/618#discussion_r27928868>
> :
>
> > @@ -1127,7 +1046,7 @@ - (BOOL)isVideoPressInfoRequestScheme:(NSString *)scheme
> >
> >  - (BOOL)isSelectionStyleScheme:(NSString*)scheme
> >  {
> > -	static NSString* const kCallbackScheme = @"callback-video-fullscreen";
> > +	static NSString* const kCallbackScheme = @"callback-selection-style";
>
> Is this related to the issue being solved?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wordpress-mobile/WordPress-Editor-iOS/pull/618/files#r27928868>
> .
>